sos/plugins/ipvs.py

```diff
@@ -21,14 +21,15 @@ class Ipvs(Plugin, RedHatPlugin, DebianPlugin):
     packages = ('ipvsadm',)
 
     def setup(self):
-        self.add_cmd_output([
-            "ipvsadm -Ln",
-            "ipvsadm -Ln --connection",
-            "ipvsadm -Ln --persistent-conn",
-            "ipvsadm -Ln --rate",
-            "ipvsadm -Ln --stats",
-            "ipvsadm -Ln --thresholds",
-            "ipvsadm -Ln --timeout"
-        ])
+        if self.is_module_loaded('ip_vs'):
```

Comment:
As of 3.7 we have a new `SoSPredicate` feature that should be used for gating like this - it allows us to still have a history of when we don't run commands when the condition isn't met.

E.G.
```python
self.add_cmd_output([
     "ipvsadm -Ln",
     "ipvsadm -Ln --connection",
     "ipvsadm -Ln --persistent-conn",
     "ipvsadm -Ln --rate",
     "ipvsadm -Ln --stats",
     "ipvsadm -Ln --thresholds",
     "ipvsadm -Ln --timeout"
], pred=SoSPredicate(self, modules=['ip_vs]'))
```